tasks/system_package_management.yml

```diff
@@ -0,0 +1,70 @@
+---
+# .. vim: foldmarker=[[[,]]]:foldmethod=marker
+
+- name: Get ownCloud APT OpenPGP key
+  apt_key:
+    url: '{{ owncloud__apt_repo_key_url }}'
```

Comment:
Thanks for the review! I think I already tried that a few months ago and just retired. The public key is not on keyservers it seems: gpg --recv-keys BCECA90325B072AB1245F739AB7C32C35180350A

I don’t see a direct problem there. Would you prefer to include the public key with the role?